app/models/data.py

```diff
@@ -269,7 +269,7 @@ def totalTestResults(self):
                 return self.negative or 0
```

Comment:
The `or 0` up here is intentional, since the behavior for posNeg is different than for all real columns. This seems to match what the public sheet is doing now.